templatestore/frontend/package.json

```diff
@@ -32,10 +32,12 @@
     "axios": "^0.19.2",
     "compression-webpack-plugin": "^4.0.1",
     "file-loader": "^5.0.2",
+    "promise": "^8.1.0",
```

Comment:
I can't see this being used anywhere. if so, please uninstall using npm uninstall promise